DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -1,162 +1,162 @@
-            return (adminUserWithActiveCentre, delegateUsersWithActiveCentres);
-        }
-
-        public List<CentreUserDetails> GetUserCentres(AdminUser? adminUser, List<DelegateUser> delegateUsers)
-        {
-            var availableCentres = delegateUsers
-                .Select(du =>
-                    new CentreUserDetails(du.CentreId, du.CentreName, adminUser?.CentreId == du.CentreId, true))
-                .ToList();
-
-            if (adminUser != null && availableCentres.All(c => c.CentreId != adminUser.CentreId))
-            {
-                availableCentres.Add(new CentreUserDetails(adminUser.CentreId, adminUser.CentreName, true));
-            }
-
-            return availableCentres.OrderByDescending(ac => ac.IsAdmin).ThenBy(ac => ac.CentreName).ToList();
-        }
-
-        public bool TryUpdateUserAccountDetails
-        (
-            int? adminId,
-            int? delegateId,
-            string password,
-            string firstName,
-            string surname,
-            string email,
-            byte[]? profileImage
-        )
-        {
-            var (verifiedAdminUser, verifiedDelegateUsers) =
-                GetVerifiedLinkedUsersAccounts(adminId, delegateId, password);
-
-            if (verifiedAdminUser == null && verifiedDelegateUsers.Count == 0)
-            {
-                return false;
-            }
-
-            if (verifiedAdminUser != null)
-            {
-                userDataService.UpdateAdminUser(firstName, surname, email, profileImage, verifiedAdminUser.Id);
-            }
-
-            if (verifiedDelegateUsers.Count != 0)
-            {
-                var delegateIds = verifiedDelegateUsers.Select(d => d.Id).ToArray();
-                userDataService.UpdateDelegateUsers(firstName, surname, email, profileImage, delegateIds);
-            }
-
-            return true;
-        }
-
-        private (AdminUser?, List<DelegateUser>) GetVerifiedLinkedUsersAccounts
-        (
-            int? adminId,
-            int? delegateId,
-            string password
-        )
-        {
-            var (loggedInAdminUser, loggedInDelegateUser) = GetUsersById(adminId, delegateId);
-
-            var signedInEmail = loggedInAdminUser?.EmailAddress ?? loggedInDelegateUser?.EmailAddress;
-
-            if (signedInEmail == null)
-            {
-                return (null, new List<DelegateUser>());
-            }
-
-            var (adminUser, delegateUsers) = GetUsersByEmailAddress(signedInEmail);
-
-            return loginService.VerifyUsers(password, adminUser, delegateUsers);
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public interface IUserService
+    {
+        public (AdminUser? adminUser, List<DelegateUser> delegateUsers) GetUsersByUsername(string username);

```

Comment:
Is there a particular reason why these outputs now have names in the interface definition? 

Also it looks like line endings have been changed on this file - can we check that the line endings are correct, as it looks like this UserService file already has CRLF on master.

Reply:
They have names to make it easier to use them elsewhere - if you only want the adminUser you can do `GetUsersByUsername.adminUser` instead of `GetUsersByUsername.item1` which is much nicer. I had to do something like this so I updated all the interface definitions.

Reply:
The line endings is weird, if I check out current master they're CRLF but I branched off from `32db337b9cdc5aa91a7f33fe9559050257bc805c` which has them as LF for me at least.